### DISCUSSION TITLE
when I build ImageMagick in mingw,  I met  a compile error...

### DISCUSSION THREAD

**Discussion by xiaoxiongli** [ImageMagick/ImageMagick#4916] [Help]
Hi All:

when I build ImageMagick in mingw,  I met  a compile error...

please help me...

$ make
D:/software/mingw64/bin/make  all-am
make[1]: Entering directory 'D:/transcode/magick/magick'
  CC       utilities/magick.o
  CC       MagickCore/libMagickCore_7_Q16HDRI_la-accelerate.lo
**/usr/bin/sh: C:/Program: No such file or directory**
make[1]: *** [Makefile:8638: MagickCore/libMagickCore_7_Q16HDRI_la-accelerate.lo] Error 127
make[1]: Leaving directory 'D:/transcode/magick/magick'
make: *** [Makefile:6164: all] Error 2

**Comment by snibgo** [comment]
Please don't multi-post.
